### PR TITLE
Add "Focus Area" highlight and ribbon

### PR DESCRIPTION
**Pull request by joshlory**
Highlight a user's PLC focus areas with a blue outline and ribbon:

focus_area

### CODE REVIEW

File: code-studio/src/js/components/progress/course_progress.jsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ import color from '../../color';
 const styles = {
   flexHeader: {
     padding: '5px 10px',
-    margin: '10px 0 0 0',
+    margin: '20px 0 0 0',
```
Comment (mehalshah) [inline code-studio/src/js/components/progress/course_progress.jsx:11]: no need to specify units

Comment (joshlory) [inline code-studio/src/js/components/progress/course_progress.jsx:11]: Units are needed when using the combined shorthand.

File: dashboard/app/helpers/users_helper.rb [hunk 2]
```diff
@@ -82,6 +82,13 @@ def summarize_trophies(script, user = current_user)
       user_data[:trophies] = summarize_trophies(script, user)
     end
 
+    if script.professional_learning_course
+      unit_assignment = Plc::EnrollmentUnitAssignment.find_by(user: user, plc_course_unit: script.plc_course_unit)
+      if unit_assignment
+        user_data[:focusAreas] = unit_assignment.plc_module_assignments.map{ |a| a.plc_learning_module.stage.position }
```
Comment (mehalshah) [inline dashboard/app/helpers/users_helper.rb]: So it's a little weird to me that you are passing an array of focus area positions and calling them focus areas.

Suggest you either call the variable :focusAreaPositions. Or pass a list of stage names and change the way you are determining what is a focus area in course progress.

Comment (joshlory) [inline dashboard/app/helpers/users_helper.rb]: Ah good call, that makes sense.  Renamed `focusAreas` -> `focusAreaPositions` in 1ad78c8d1599de2004d6993570fbaae664a17fa7.

**Comment by mehalshah** [comment]
lgtm

**Pull request merged**